Sources/MediaViewer/UI/PreviewController.swift

```diff
@@ -66,6 +66,29 @@ open class PreviewController: UIViewController {
     
     public required init?(coder: NSCoder) { fatalError() }
     
+    open override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return [.portrait, .landscapeLeft, .landscapeRight]
+    }
+    
+    open override var shouldAutorotate: Bool {
+        return true
+    }
+    
+    open override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        setOrientation(to: .portrait)
+    }
+    
+    func setOrientation(to orientation: UIInterfaceOrientation) {
```

Comment:
Does this needs to be public? Should it be in some util class like Device.swift?

Reply:
made private, but I see no reason to make it a util in this package. it's specific to this controller which is the top container of preview and not used elsewhere, nor will it be.